src/platform/macos/system/surface.rs

```diff
@@ -196,6 +196,7 @@ impl Device {
         transaction::set_disable_actions(true);
 
         let superlayer = CALayer::new();
+        let _: () = msg_send![superlayer.id(), retain];
```

Comment:
Does `CALayer::new` return a CALayer with a reference? If so, won't this leak the CALayer? If that's the case, maybe the layer reference count increase should happen in the interface between surfman and winit.

Reply:
Yeah, that could happen if the native widget is from other sources and they don't clean up their window entirely like winit.
How about a platform-specific method like `retain_layer`? In this case, I can call it for winit usage.

Reply:
I think this might be fixed by https://github.com/servo/core-foundation-rs/pull/546.

Reply:
Ah thanks! I'll track the coacoa release and close this one.